arch/arm/soc/nxp_imx/rt/Kconfig.defconfig.mimxrt1052

```diff
@@ -18,6 +18,24 @@ config CLOCK_CONTROL_MCUX_CCM
 
 endif # CLOCK_CONTROL
 
+config INIT_ARM_PLL
```

Comment:
for INIT_ARM_PLL, INIT_SYS_PLL, and INIT_USB1_PLL -- is the thinking these change per SoC or per board?

Reply:
Per SoC. Certain PLL might not exist on certain SoC.

Reply:
If per SoC than instead of putting these in the defconfig put them in arch/arm/soc/nxp_imx/rt/Kconfig.soc as 'select INIT_ARM_PLL', etc. (for the one's that are bools).